app/models/receipt.rb

```diff
@@ -6,8 +6,10 @@ class Receipt < ApplicationRecord
   enum channel: { sms: "sms", mail: "mail", webhook: "webhook" }, _prefix: :channel
 
   # Relations
-  belongs_to :rdv  # We could reference a RdvsUser directly (aka a “Participation”) but this would not work for responsible users of relatives.
-  belongs_to :user # Moreover, if we remove a user for a Rdv (deleting the RdvsUser), we still want to keep the receipt.
+  # We could reference a RdvsUser directly (aka a “Participation”) but this would not work for responsible users of relatives.
+  # Moreover, if we remove a user for a Rdv (deleting the RdvsUser), we still want to keep the receipt.
+  belongs_to :rdv
+  belongs_to :user, optional: true
```

Comment:
Comme l'explique le commentaire, parfois le `receipt` n'a pas de `user`, donc il faut déclarer `optional: true` car par défaut `belongs_to` [valide la présence](https://www.bigbinary.com/blog/rails-5-makes-belong-to-association-required-by-default).

Reply:
Je ne suis pas sûr de comprendre cette partie. Dans un receipt il y a toujours un user non ?

Reply:
Je ne suis pas sûr de comprendre cette partie.

Il me semblait que dans un receipt il y a toujours un user, peu importe les jeux de responsable ou pas... Je préfère bloquer la PR en attendant ton retour pour en parler.

Reply:
Effectivement :-/

Merci !